### PR TITLE
[IMP] web/all: move the active/unarchive in the action button

### PR DESCRIPTION
**Pull request by Gorash**
In an onborading, poeple click on "Archive" -> they don't find the task
anymore. Remove "Active" button from the stat bar and move it in the
"Action" (only in form). The label is "Archive". When it is Archived,
the button become "Restore"

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/kanban/kanban_controller.js [hunk 1]
```diff
@@ -207,7 +208,16 @@ var KanbanController = BasicController.extend({
...
-                    kanban_record.update(data);
+                    var domain = self.renderer.state.domain;
+                    if ('active' in data.data && _.pluck(domain, 0).indexOf('active') === -1) {
+                        domain = [['active', '=', true]].concat(domain);
+                    }
+                    var visible = new Domain(domain).compute(data.evalContext);
+                    if (visible) {
+                        kanban_record.update(data);
+                    } else {
+                        kanban_record.destroy();
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/kanban/kanban_controller.js]: this is not enough, it does not update the model

Comment (ged-odoo) [reply addons/web/static/src/js/views/kanban/kanban_controller.js]: something more should be done, updating the model, somewhow

File: addons/web/static/src/js/views/kanban/kanban_controller.js [hunk 2]
```diff
@@ -207,7 +208,16 @@ var KanbanController = BasicController.extend({
                 self.model.reload(record.id).then(function (db_id) {
                     var data = self.model.get(db_id);
                     var kanban_record = event.target;
-                    kanban_record.update(data);
+                    var domain = self.renderer.state.domain;
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/kanban/kanban_controller.js]: it would be better to get the domain in the data, not in the renderer state

**Comment by Gorash** [comment]
@ged-odoo Can you review? thanks

**Comment by Gorash** [comment]
@ged-odoo I rebase it

**Comment by rdeodoo** [comment]
This was recently done in master right? Can we close?

**Pull request closed without merging**